### PR TITLE
[WIP] IOS sdk

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build

**Comment by vargaz** [comment]
@monojenkins build

File: builds/Makefile [hunk 1]
```diff
@@ -1792,71 +1792,39 @@ $(BUILD_DESTDIR)/targettv/tmp-lib/Mono: $(BUILD_DESTDIR)/targettv/lib/libmonosge
...
-	$(MAKE) -C llvm64 all install
-	$(Q) touch .stamp-configure-llvm64
-
-clean-llvm:
-	-rm -rf llvm .stamp-configure-llvm .stamp-build-llvm llvm.config.cache
-clean-llvm64:
-	-rm -rf llvm64 .stamp-configure-llvm64 .stamp-build-llvm64 llvm64.config.cache
+clean-llvm: $(SDK_CONFIG)
+	$(MAKE) -C $(MONO_PATH)/sdks/builds clean-ios-llvm
+	$(RM) .stamp-*-llvm*
```
Comment (rolfbjarne) [inline builds/Makefile]: `RM` isn't defined, so this would try to execute `-stamp-*-llvm*`.

`$(Q) rm -f .stamp-*-llvm*` should work fine though.

Comment (vargaz) [reply builds/Makefile]: RM is predefined make variable, its set to rm -rf.

**Comment by vargaz** [comment]
build

**Comment by vargaz** [comment]
@monojenkins build

**Comment by vargaz** [comment]
@monojenkins build

**Comment by vargaz** [comment]
Closed in favor of https://github.com/xamarin/xamarin-macios/pull/3640.

**Pull request closed without merging**